scripts/docker_version_control.sh

```diff
@@ -21,7 +21,7 @@ tag=`echo $image_tag | cut -f2 -d:`
 
 if [[ ",$SONIC_VERSION_CONTROL_COMPONENTS," == *,all,* ]] || [[ ",$SONIC_VERSION_CONTROL_COMPONENTS," == *,docker,* ]]; then
     # if docker image not in white list, exit
-    if [[ "$IMAGENAME" != sonic-slave-* ]] && [[ "$IMAGENAME" != docker-base* ]] && [[ "$IMAGENAME" != debian:* ]] && [[ "$IMAGENAME" != multiarch/debian-debootstrap:* ]];then
+    if [[ "$image_tag" != sonic-slave-* ]] && [[ "$image_tag" != docker-base* ]] && [[ "$image_tag" != debian:* ]] && [[ "$image_tag" != multiarch/debian-debootstrap:* ]];then
```

Comment:
Do we support amd64/debian:buster, arm64v8/debian:buster, arm32v7/debian:buster?
Is it a break change of docker image reproducible build in below PR?
https://github.com/Azure/sonic-buildimage/pull/7731/files

Reply:
1. we should wait to get these dockers' sha256 value. after that we can add them to white list.
2. The main issue is git script. This is a potential issue.